internal/http/services/owncloud/ocdav/propfind.go

```diff
@@ -816,6 +816,14 @@ func (s *svc) mdToPropResponse(ctx context.Context, pf *propfindXML, md *provide
 	return &response, nil
 }
 
+// be defensive about wrong encoded etags
+func quoteEtag(etag string) string {
+	if strings.HasPrefix(etag, "W/") {
+		return `W/"` + strings.Trim(etag[2:], `"`) + `"`
```

Comment:
This will panic when etag is empty.

Reply:
no, see line above: `if strings.HasPrefix(etag, "W/") {`